src/locales/en.js

```diff
@@ -46,6 +46,14 @@ export default {
     'widget.page.create.title.it.placeholder': 'Italian title',
     'widget.page.create.group': 'Group',
     'widget.page.tab.customUi': 'Custom UI',
+    'widget.list.title': 'Widget',
```

Comment:
Why this file contains less additions than the it.js ?
Please align the files, or remove unneeded labels